src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "name": "Google BigQuery API",
+    "documentation": "https://cloud.google.com/bigquery/docs/reference/libraries",
+    "description":"Google's fully managed, petabyte scale, serverless data warehouse for analytics",
+    "transport": "http",
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/bigquery.svg",
+    "clients": [
+      {
+      "language":"java",
+      "site":"https://cloud.google.com/bigquery/docs/reference/libraries#client-libraries-install-java",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/0.22.0/apidocs/com/google/cloud/bigquery/package-summary.html",
+      "version" : "0.22.0",
+      "infotip":"The com.google.cloud.bigquery packages",
+      "status" : "beta",  
+      "source" : "https://github.com/GoogleCloudPlatform/google-cloud-java/tree/master/google-cloud-bigquery",
+      "languageLevel" : "1.7.0",
+      "mavenCoordinates" : {
+         "groupId" : "com.google.cloud",
+         "artifactId" : "google-cloud-bigquery",
+         "version" : "0.22.0-beta"
+      },
+      "compatible" : [
+          "Android",
```

Comment:
We could simply drop the platforms section for now. We can always put it back in later if we discover a need for it. What do you think?

That said, I think Java 7 should work in all environments including Android for HTTP transport based protocols per https://github.com/GoogleCloudPlatform/google-cloud-java#supported-platforms
 
"Clients in this repository use either HTTP or gRPC for the transport layer. All HTTP-based clients should work in all environments."